conda_forge_tick/migrators/migration_yaml.py

```diff
@@ -127,3 +133,142 @@ def order(
         return sorted(
             graph, key=lambda x: (len(nx.descendants(total_graph, x)), x), reverse=True,
         )
+
+
+class MigrationYamlCreator(Migrator):
+    """Migrator creating migration yaml files."""
+
+    migrator_version = 0
+    rerender = False
+
+    # TODO: add a description kwarg for the status page at some point.
```

Comment:
IDK if this needs to land on the status page since it will only ever have one PR issued.